data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -289,6 +290,7 @@ private void addV1ValidatorHandlers(final DataProvider dataProvider) {
         tech.pegasys.teku.beaconrestapi.handlers.v1.validator.GetNewBlock.ROUTE,
         new tech.pegasys.teku.beaconrestapi.handlers.v1.validator.GetNewBlock(
             dataProvider, jsonProvider));
+    app.get(GetAttestationData.ROUTE, new GetAttestationData(dataProvider, jsonProvider));
```

Comment:
There's a BeaconRestApiV1Test that just checks that endpoints are plugged in, might be worth adding to that for the new endpoint.

Reply:
Ah, I added it to `BeaconRestApiTest`.  Will move it.